frontend/document/credocument.lua

```diff
@@ -1212,6 +1226,7 @@ function CreDocument:setFontSize(new_font_size)
     if new_font_size then
         logger.dbg("CreDocument: set font size", new_font_size)
         self._document:setFontSize(new_font_size)
+        self:setNormalLineHeight(G_reader_settings:nilOrTrue("cre_size_based_normal_line_height"))
```

Comment:
Feels a little odd to pass one of the 2 settings as arg, and use the other one inside the called function.